### ISSUE TITLE
application needs to be indexable by google

### ISSUE THREAD

**Issue by abrin** [openskope/SKOPE#10]
One of the issues critical to our success is the ability of potential users to find the SKOPE tool and visualizations.  This will require that not only does the homepage and general website be indexable, but ideally the app itself is accessible and indexable by a search engine.  This probably means adding "[spiderable](https://blog.meteor.com/search-engine-optimization-44301455c6b5)" to the app, as well as looking at "pre-rendering" and other issues around search engine optimization.

**Comment by tmcphillips** [comment]
Can you provide some specific examples of search strings, what the search results would look like in Google or other search engine, the pages these serach results would link to in the SKOPE application, and what would be displayed on those pages when someone follows the links?

**Comment by abrin** [comment]
Not sure we're ready for that yet. 

But, is it reasonable to assert that:
* we want people to find the  homepage in google?
* we want people to be able to discovery models in google?


On Wed, Dec 6, 2017, at 01:08 PM, Timothy McPhillips wrote:
> Can you provide some specific examples of search strings, what the
> search results would look like in Google or other search engine, the
> pages these serach results would link to in the SKOPE application, and
> what would be displayed on those pages when someone follows the links?> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].> 


Links:

  1. https://github.com/openskope/SKOPE/issues/10#issuecomment-349759409

**Comment by tmcphillips** [comment]
Definitely.  I wonder if a Google search result for "PaleoCAR" would link to a page on the SKOPE web site describing PaleoCAR and listing the official run results which would then link to map visualizations in the SKOPE application; or rather to the search page in the SKOPE application showing the results for a search for PaleoCAR results?

By the way, the top result when searching for "prism precipitation" on Google looks like this:

![googleforprismpreciptation](https://user-images.githubusercontent.com/3218259/33683775-e6d49fe2-da80-11e7-8d13-c75054cc1c1d.PNG)

**Comment by abrin** [comment]
I think moving forward we should:

A) Make sure that the Meteor App can be indexed
B) identify pages that should be indexable
C) Figure out how we want results to look

My only point in creating the ticket was that Javascript is not easily
indexed by all or most search engines (though google does an amazing job
at trying to).


On Wed, Dec 6, 2017, at 01:32 PM, Timothy McPhillips wrote:
> Definitely.  I wonder if a Google search result for "PaleoCAR" would
> link to a page on the SKOPE web site describing PaleoCAR and listing
> the official run results which would then link to map visualizations
> in the SKOPE application; or rather to the search page in the SKOPE
> application showing the results for a search for PaleoCAR results?> By the way, the top result when searching for "prism precipitation" on
> Google looks like this:> googleforprismpreciptation


> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].> 


Links:

  1. https://github.com/openskope/SKOPE/issues/10#issuecomment-349765743
